src/TaylorN/TaylorN.jl

```diff
@@ -6,14 +6,15 @@ degree(f::TaylorN) = f.order
 setindex!(f::TaylorN, x, i) = f.coeffs[i+1] = x
 
 doc"""
-A `Taylor1Model` represents a polynomial approximation to a function $f(t)$ of a single variable.
+A `TaylorNModel` represents a polynomial approximation to a function $f(t)$ of `N` independent variables.
```

Comment:
I know you just updated, so it's not your fault, but I dont think that is correct, I would say a taylor model is a rigorous approximation of a function composed by a polynomial approximation **and** an interval bound on the error